website/en/docs/types/objects.md

```diff
@@ -215,7 +215,7 @@ method({
 > **Note:** This is because of ["width subtyping"](../../lang/width-subtyping/).
 
 Sometimes it is useful to disable this behavior and only allow a specific set
-of properties. For this, Flow supports "exact" object types.
+of properties. For this, Flow supports "exact" object types through enabling to add a pair of vertical bars.
```

Comment:
Oh, sad this didn't get more eyes, I like this change! I would change this to 

> Flow supports "exact" object types. Their syntax is just like that for objects, but with vertical bars after the opening curly brace and before the closing one:

or something along those lines